src/platform/silabs/rs911x/BLEManagerImpl.cpp

```diff
@@ -247,8 +247,10 @@ namespace {
 #define BLE_CONFIG_MAX_CE_LENGTH (0xFFFF) // Leave to max value
 
 #define BLE_DEFAULT_TIMER_PERIOD_MS (1)
```

Comment:
shouldn't these be in the header file?

Reply:
Header file is common for both efr32 and rs911x so keeping it here

Reply:
```cpp
#if (SLI_SI91X_ENABLE_BLE || RSI_BLE_ENABLE)
#define BLE_MIN_CONNECTION_INTERVAL_MS 45 // 45 msec
#define BLE_MAX_CONNECTION_INTERVAL_MS 45 // 45 msec
#define BLE_SLAVE_LATENCY_MS 0
#define BLE_TIMEOUT_MS 400
#endif // (SLI_SI91X_ENABLE_BLE || RSI_BLE_ENABLE)
```

I see this code there, so I guess this can be added to this, or this code can be moved from there to here

Reply:
updated